src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -78,7 +82,8 @@
               (let [pairs (core/partition 2 v1)
                     names (map first pairs)
                     gens (map second pairs)]
-                `(for [[~@names] (gen/tuple ~@gens)
+                `(for [[~@names]
+                       (~'gen/tuple ~@gens)
```

Comment:
but is it true that if test.check had unified namespace names the problem would go away?